charts/datadog/templates/NOTES.txt

```diff
@@ -155,24 +155,14 @@ The Datadog Agent is listening on port {{ $apmPort }} for APM service.
 
 {{- end }}
 
-{{- if and .Values.datadog.apm.instrumentation.enabled_namespaces (eq (include "cluster-agent-enabled" .) "false")}}
+{{- if and .Values.datadog.apm.instrumentation.enabledNamespaces (not .Values.datadog.apm.instrumentation.enabled) }}
```

Comment:
This was the typo in the initial version of the check. The configuration name was always `datadog.apm.instrumentation.enabledNamespaces`.